core/collateral/checkpoint.go

```diff
@@ -130,11 +130,12 @@ func (e *Engine) getCheckpointBalances() []*checkpoint.AssetBalance {
 		}
 		switch acc.Type {
```

Comment:
Shouldn't the holding account be added here somehow?

Reply:
it should